File: src/main/java/cn/lnkdoc/sdk/uia/instance/bjtoon/domain/AccessToken.java
package cn.lnkdoc.sdk.uia.instance.bjtoon.domain;

import com.alibaba.fastjson2.annotation.JSONField;

import java.io.Serial;
import java.io.Serializable;

/**
 *
 * AccessToken Object to be returned
 *
 * @author qinyi
 * @version  v 1.0 2018-10-23
 */
@SuppressWarnings(value = {"unused"})
public class AccessToken implements Serializable {
	@Serial
	private static final long serialVersionUID = -508722906865178L;
	/**
	 * accessToken
	 */
	private String accessToken;
	/**
	 * tokenType
	 */
	private String tokenType;
	/**
	 * expiresIn
	 */
	private String expiresIn;
	/**
	 * refreshToken
	 */
	private String refreshToken;

	/**
	 * getter
	 * 
	 * @return accessToken
	 */
	public String getAccessToken() {
		return accessToken;
	}

	/**
	 * setter
	 * 
	 * @param accessToken  accessToken
	 */
	@JSONField(name = "access_token")
	public void setAccessToken(String accessToken) {
		this.accessToken = accessToken;
	}

	/**
	 * getter
	 *
	 * @return tokenType
	 */
	public String getTokenType() {
		return tokenType;
	}

	/**
	 * setter
	 * 
	 * @param tokenType  tokenType
	 */
	@JSONField(name = "token_type")
	public void setTokenType(String tokenType) {
		this.tokenType = tokenType;
	}

	/**
	 * getter
	 *
	 * @return expiresIn
	 */
	public String getExpiresIn() {
		return expiresIn;
	}

	/**
	 * setter
	 * 
	 * @param expiresIn expiresIn
	 */
	@JSONField(name = "expires_in")
	public void setExpiresIn(String expiresIn) {
		this.expiresIn = expiresIn;
	}

	/**
	 * getter
	 *
	 * @return refreshToken
	 */
	public String getRefreshToken() {
		return refreshToken;
	}

	/**
	 * setter
	 * 
	 * @param refreshToken refreshToken
	 */
	@JSONField(name = "refresh_token")
	public void setRefreshToken(String refreshToken) {
		this.refreshToken = refreshToken;
	}

}

File: src/main/kotlin/cn/lnkdoc/sdk/uia/instance/bjtoon/domain/AccessToken.kt
package cn.lnkdoc.sdk.uia.instance.bjtoon.domain

import com.alibaba.fastjson2.annotation.JSONField
import java.io.Serializable


/**
 *
 * AccessToken Object to be returned
 *
 * @author qinyi
 * @version  v 1.0 2018-10-23
 */
@Suppress("unused")
open class AccessToken : Serializable {
    /**
     * accessToken
     */
    @set:JSONField(name = "access_token")
    @get:JSONField(name = "accessToken")
    var accessToken: String? = null
    /**
     * tokenType
     */
    @set:JSONField(name = "token_type")
    var tokenType: String? = null
    /**
     * expiresIn
     */
    @set:JSONField(name = "expires_in")
    var expiresIn: String? = null
    /**
     * refreshToken
     */
    @set:JSONField(name = "refresh_token")
    var refreshToken: String? = null
    
    companion object {
        private const val serialVersionUID = -508722906865178L
    }
}